code/Commenting.php

```diff
@@ -40,6 +40,8 @@ class Commenting {
 		'html_allowed' => false, // allow for sanitized HTML in comments
 		'html_allowed_elements' => array('a', 'img', 'i', 'b'),
 		'use_preview' => false, // preview formatted comment (when allowing HTML). Requires include_js=true
+		'thread_comments' => false, // allow replies to existing comments.  Requires include_js=true,
```

Comment:
We really need to develop a non-js compatible reply mechanism as well; Not everyone will want to cater to js-only enabled environments.